manifests/overlays/prod/applications/data_hub/dh-stage-logstash.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: dh-stage-logstash
+spec:
+  destination:
+    namespace: dh-stage-ingest
+    server: https://paas.stage.psi.redhat.com:443
+  project: data-hub
+  source:
+    path: logstash/overlays/stage
+    repoURL: https://github.com/AICoE/internal-data-hub.git
+    targetRevision: master
+  syncPolicy:
+    automated:
```

Comment:
Completely Optional: 
I'd suggest keeping auto-sync off _for now_ while you verify everything works as intended, then send a pr turning it on.

Reply:
This is on stage, so I think autosync is fine there. For prod we should definitely leave auto sync off for now.